spl/Cargo.toml

```diff
@@ -23,7 +23,8 @@ metadata = ["mpl-token-metadata"]
 mint = []
 stake = ["borsh"]
 token = ["spl-token"]
-token_2022 = ["spl-token-2022"]
+token_2022 = ["spl-token-2022", "borsh"]
+token_2022_extensions = ["spl-token-2022", "borsh", "spl-token-group-interface", "spl-token-metadata-interface", "spl-pod"]
```

Comment:
Can we remove `borsh` if the 2 functions that we use start taking in raw arguments i.e. without creating a `borsh` struct?